source/collimation.f90

```diff
@@ -2905,7 +2922,7 @@ subroutine collimate_do_collimator(stracki)
       else !if(n_slices.gt.one .and. totals.gt.smin_slices .and. totals.lt.smax_slices .and.
 !     Treatment of non-sliced collimators
 
-#ifdef G4COLLIMAT
+#ifdef G4COLLIMATION
 !! Add the geant4 geometry
         if(firstrun.and.iturn.eq.1) then
```

Comment:
Not really, but it is still used everywhere, perhaps it can be removed in the future. I would say it is best to do in one go throughout the entire file.